services/clusters-keeper/src/simcore_service_clusters_keeper/modules/remote_debug.py

```diff
@@ -0,0 +1,26 @@
+""" Setup remote debugger with debugpy - a debugger for Python
+    https://github.com/microsoft/debugpy
+
+"""
+import logging
+
+from fastapi import FastAPI
+
+_logger = logging.getLogger(__name__)
+_REMOTE_DEBUGGING_PORT = 3000
+
+
+def setup_remote_debugging(app: FastAPI) -> None:
+    def on_startup() -> None:
+        try:
+            _logger.info("Attaching debugpy on %s...", _REMOTE_DEBUGGING_PORT)
+
+            import debugpy
```

Comment:
Very nice! This is linked with https://github.com/ITISFoundation/osparc-simcore/issues/2844. Please add in related issues.

Q1: can we add this functionality as CLI options (i.e. in the boot)
Using it as a "plugin" has the following disadvantages:
- cannot debug anything in the startup (since it iwll not be possible until plugin is enables the debugging)
- it is code intrusive

Q2: it would be nice to have this also in release mode. 
Having a boot mode that installs debugpy and spawns the debug server via CLI would be ideal

Reply:
I guess there is nothing against that. nevertheless I will postpone doing this to some better time. I will move these questions in the aforementioned issue